src/components/ComingSoon/ComingSoon.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <section>
+    <Heading class="title" size="large" weight="medium">{{ title }}</Heading>
+    <div class="text-info">
+      <TextParagraph class="description" size="medium"> {{ description }}</TextParagraph>
+    </div>
+    <div class="image-wrapper">
+      <img :src="image" alt="Sheep working" width="360" height="264">
+    </div>
+    <Button :data-testid="dataTestId" @click="onClose">
+      Aceptar
```

Comment:
This button will also need to be localized, so please refactor it to come from a new `closeButtonText` prop for this component (and update the Storybook story to add a knob for that prop as well).